x-pack/test/security_solution_api_integration/test_suites/genai/knowledge_base/entries/trial_license_complete_tier/configs/ess.config.ts

```diff
@@ -16,11 +16,38 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     ...functionalConfig.getAll(),
     kbnTestServer: {
       ...functionalConfig.get('kbnTestServer'),
-      serverArgs: [...functionalConfig.get('kbnTestServer.serverArgs')],
+      serverArgs: [
+        ...functionalConfig
+          .get('kbnTestServer.serverArgs')
+          // ssl: false as ML vocab API is broken with SSL enabled
+          .filter(
+            (a: string) =>
+              !(
+                a.startsWith('--elasticsearch.hosts=') ||
+                a.startsWith('--elasticsearch.ssl.certificateAuthorities=')
+              )
+          ),
```

Comment:
As noted in the description, we must run with `ssl:false` for certain ml API's to function. See https://github.com/elastic/kibana/issues/193477 for details.